### DISCUSSION TITLE
Pikachu SSD

### DISCUSSION THREAD

**Discussion by SiebeF** [deepjavalibrary/djl#1779] [Q&A] [answered]
The tutorial of how to train the SSD-model for the pikachu dataset works, only when I want to use the predict function it seems to go wrong, I just copied the code from the site (see image).
![image](https://user-images.githubusercontent.com/92974924/177790577-0b528f5e-1afe-4cca-9f29-8f9bd22c7d3d.png)
I get this:
![image](https://user-images.githubusercontent.com/92974924/177790787-b7ede110-d8eb-46c7-8306-0722baaab358.png)
and in IDE this:
![image](https://user-images.githubusercontent.com/92974924/177790933-f89c9a19-101b-42c4-81fd-398b72558267.png)
I hope somebody can help me with this, I'm just beginning with the DJL-library, thanks!

**Comment by zachgk** [comment, marked as answer]
So, it looks like you have two different problems. The above problem is because you are putting that code into your terminal. The terminal expects shell code and you are passing it java code. The syntax error means that it doesn't even know what the code is trying to say because it is the wrong kind of code.

The second example in the IDE is better. The red line under predict is part of Java's checked exceptions feature. Basically, the predict function can throw some exceptions if something goes wrong and you need to either handle or ignore the exceptions. Search for "Java checked exception" and you should find some more details about how this works. You can also get suggestions from your IDE about how to deal with the exceptions
